packages/patternfly-4/react-core/src/components/Tooltip/Tooltip.md

```diff
@@ -2,10 +2,10 @@
 title: 'Tooltip'
 cssPrefix: 'pf-c-tooltip'
 typescript: true
-propComponents: ['Tooltip']
+propComponents: ['Tooltip', 'TooltipContent']
```

Comment:
Good catch, we export `TooltipContent` in `Tooltip/index.ts` but don't show its props in the docs.

Edit: We should add a followup issue to provide examples on how/why to use `TooltipContent`.

Reply:
All set! #3088